openmrs-19/src/test/java/org/motechproject/openmrs19/tasks/impl/OpenMRSActionProxyServiceTest.java

```diff
@@ -75,15 +79,20 @@ public void shouldCreateEncounterWithGivenParameters() {
         Person person = new Person();
         person.setUuid("30");
         provider.setPerson(person);
+        
+        DateTime encounterDatetime = new DateTime("2000-08-16T07:22:05Z");
+        Map<String, String> observations = new HashMap();
+        observations.put("1","1");
+        List<Observation> obsList = convertObservationMapToList(observations, encounterDatetime);
```

Comment:
please just create observation list using known values, maybe add constructors to appropriate classes.